src/ingest_llm_as/services/omegakg_client.py

```
@@ -0,0 +1,130 @@
+
+logger = logging.getLogger(__name__)
+
+class OmegaKGClient:
+    """
+    HTTP client for OmegaKG validation API.
+    
+    Handles authentication, retries, and error handling for
+    structured knowledge digest submission.
+    """
+    
+    def __init__(self):
```

Ruff (0.14.11)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)